contracts/RelayHub.sol

```diff
@@ -140,9 +142,10 @@ contract RelayHub is RelayHubApi {
     }
 
     function remove_relay_internal(address relay) internal {
-//        delete relays[relay];
+        delete relays[relay];
```

Comment:
either delete or set fields to zero, not both
(and if setting fields to zero, set them all, not only timestamp..

Reply:
done